perun/collect/trace/strategy.py

```diff
@@ -296,24 +388,26 @@ def _static_stap_extractor(binary):
     return ''
 
 
-def _static_probe_filter(static_list):
+def _static_probe_names(static_list):
```

Comment:
Suggestion: rename to something more function-like. Though I guess this is used as a filter?